packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -561,6 +561,42 @@ def test_3dt_bundle_download_and_extract(cluster):
     # for public agent host
     expected_public_agent_files = ['dcos-mesos-slave-public.service.gz'] + expected_common_files
 
+    def _read_from_zip(z: zipfile.ZipFile, item: str, to_json=True):
+        # raises KeyError if item is not in zipfile.
+        item_content = z.read(item).decode()
+
+        if to_json:
+            # raises ValueError if cannot deserialize item_content.
+            return json.loads(item_content)
+
+        return item_content
+
+    def _get_3dt_health(z: zipfile.ZipFile, item: str):
+        # try to load 3dt health report and validate the report is for this host
+        try:
+            health_report = _read_from_zip(z, item)
+        except KeyError as e:
```

Comment:
it looks like this `e` is unused, although nothing shows up in the code-quality check which weirds me out a bit.